research/flamby/fed_isic2019/fenda/run_fold_experiment.slrm

```diff
@@ -117,6 +117,7 @@ do
     nohup python -m research.flamby.fed_isic2019.fenda.server \
         --config_path ${SERVER_CONFIG_PATH} \
         --server_address ${SERVER_ADDRESS} \
+        --run_name ${RUN_NAME} \
```

Comment:
I don't think these are needed for FENDA, but I may be wrong.

Reply:
If we want to run with auxiliary task, that's where we define it. But for vanilla one we can omit it.

Reply:
Maybe I'm misunderstanding, but the fenda server doesn't appear to use this run name for anything. That is, it takes the argument, but doesn't use it.

Reply:
I think it does use it in checkpointing and saving models under that subfolder.

Reply:
Because FENDA is a personal model, it actually uses the `PersonalServer` class. These models don't save a "server-side" model. So only the client mains receive a `run_name` argument. Here you're passing it to the server main, which actually doesn't need it.

Reply:
Yeah I figured it out later and omitted it. I have added these run_name for loading from fedavg because of experiments.